kolibri/plugins/facility/assets/src/views/FacilityConfigPage/index.vue

```diff
@@ -123,12 +147,31 @@
         }
         return null;
       },
+      lastPartId() {
+        return this.facilityId.substr(this.facilityId.length - 4);
```

Comment:
I just learned we are actually using the _first_ four characters of the ID.

```suggestion
        return this.facilityId.slice(this.facilityId, 0, 4);
```

Reply:
I'll change it, but, is that logical? it's more likely the 4 last will be different than the first four

Reply:
That's my intuition too actually... could you just keep it at first four for now, and then we can change it to the last four across the whole app.

Reply:
I've left it with the first four